migrations/versions/0363a5c5c9d2_add_fields_to_run_and_benchmark_results.py

```diff
@@ -0,0 +1,41 @@
+"""add_fields_to_run_and_benchmark_results
+
+Revision ID: 0363a5c5c9d2
+Revises: 459166107c9d
+Create Date: 2022-10-20 11:14:01.529869
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "0363a5c5c9d2"
+down_revision = "459166107c9d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Do we want to backfill any of these (maybe error_type)? or perhaps it's not worth the effort

Reply:
I can backfill it when we decide on what categories we want to have. I would avoid doing this for now since we don't know 100%.